pages/linux/scanimage.md

```diff
@@ -0,0 +1,11 @@
+# scanimage
+
+> Scan images with the Scanner Access Now Easy API.
+
+- Make sure your scanner is recognized:
```

Comment:
What does this mean ?

Reply:
The operating system has to know that you have a scanner attached, and, more to the point, that it _is_ a scanner in an appropriate state. "Recognized" seems appropriate.

Reply:
I was trying to understand how does `-L` map to the description. From the man [page](https://linux.die.net/man/1/scanimage), it seems that `-L` lists the devices. Let's make that the description ?